terraform/gke/README.md

```diff
@@ -440,16 +440,26 @@ similar process.
     [Troubleshooting](#troubleshooting) section for more details.
 
     If you have chosen to use your own Spanner instance, please edit the above
-    configuration file accordingly.
+    configuration files accordingly.
 
 10.  To configure the Autoscaler and begin scaling operations, run the following
      command:
 
      ```sh
-     kubectl apply -f autoscaler-config/autoscaler-config.yaml
+     kubectl apply -f autoscaler-config/
      ```
 
-11.  Any changes made to the configuration file and applied with `kubectl
+     The sample configuration creates two schedules for scaling operations; a
+     frequently running schedule to dynamically scale the Spanner instance
+     according to utilization, and an hourly schedule to directly scale the
+     Spanner instance every hour.
+
+     Note there are some concurrency nuances when using the `DIRECT` scaling
```

Comment:
I suggest removing this paragraph.
We will address the nuances in the upcoming scenario documentation.